samza-sql/src/main/java/org/apache/samza/sql/util/SamzaSqlQueryParser.java

```diff
@@ -165,6 +165,7 @@ private static Planner createPlanner() {
         .traitDefs(traitDefs)
         .context(Contexts.EMPTY_CONTEXT)
         .costFactory(null)
+        //.programs(Programs.CALC_PROGRAM)
```

Comment:
Just curious. Why did we comment this out? If this is not necessary, then can we please remove it?